backend/Dockerfile

```diff
@@ -1,12 +1,42 @@
 FROM pennlabs/django-base:b269ea1613686b1ac6370154debbb741b012de1a-3.11
+ARG DEPENDENCY_MANAGER="uv"
 
 LABEL maintainer="Penn Labs"
 
+# Install build dependencies
+RUN apt-get update && apt-get install --no-install-recommends -y gcc libpq-dev libc-dev curl \
+    && rm -rf /var/lib/apt/lists/*
+
+# Install uv
+# Download the latest installer
+ADD https://astral.sh/uv/0.6.0/install.sh /uv-installer.sh
```

Comment:
curious why is this ur installation method of choice?

Reply:
I either misread the uv docs or it wasn't there (they seem to be pushing a lot of updates). I'm switching it to the simpler `COPY --from=ghcr.io/astral-sh/uv@sha256:2381d6aa60c326b71fd40023f921a0a3b8f91b14d5db6b90402e65a635053709 /uv /uvx /bin/`